TypeCobol.Analysis/Dfa/DfaDefPoint.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TypeCobol.Analysis.Dfa
+{
+    /// <summary>
+    /// Generic class representing a Definition Point.
+    /// </summary>
+    /// <typeparam name="I">Instruction generic Type</typeparam>
+    /// <typeparam name="V">Used Variable generic Type</typeparam>
+    public class DfaDefPoint<I, V>
```

Comment:
I think that all properties of this class should be readonly except `UserData`. This is compatible with the current usages except for the hack on SET statements for 88 levels (see [my remark](https://github.com/TypeCobolTeam/TypeCobol/pull/1751/files#r473787011) in ZCallPgmReport.cs)

Reply:
Ok for "UserData" variable, "Variable" is used by the ZCallPgmReport hack, other variables can be made internal set.
This is not the purpose here , but we can image that we can use the same hack to detect parent variable indirectly targeted by their REDEFINES or RENAMES variables.